buildflow/core/app/runtime/actors/pipeline_pattern/pull_process_push_test.py

```diff
@@ -30,11 +28,11 @@ def setup(self):
 
         @classmethod
         def source(cls):
```

Comment:
(might already be on your radar since you said tests weren't done) reminder to use the decorator syntax on this class